lizmap/modules/lizmap/lib/Server/Server.php

```diff
@@ -269,11 +269,15 @@ private function getLizmapMetadata()
         // Get Lizmap version from project.xml
         $projectInfos = AppInfos::load();
         // Version
+        $lizmapVersion = $projectInfos->version;
         $data['info'] = array();
-        $data['info']['version'] = $projectInfos->version;
+        $data['info']['version'] = $lizmapVersion;
         $data['info']['date'] = $projectInfos->versionDate;
         $data['info']['commit'] = \jApp::config()->commitSha;
 
+        $modules = new ModulesChecker();
```

Comment:
it is not necessary to call `ModulesChecker` in the class `Server`, 

If we want the `installation_complete` info in the json returned by `index.php/view/app/metadata`
, we can call the class in the controller `view/app` [controller](https://github.com/3liz/lizmap-web-client/blob/master/lizmap/modules/view/controllers/app.classic.php)